src/mlpack/methods/cf/decomposition_policies/block_krylov_svd_method.hpp

```diff
@@ -0,0 +1,173 @@
+/**
+ * @file methods/cf/decomposition_policies/block_krylov_svd_method.hpp
+ * @author Adarsh Santoria
+ *
+ * Implementation of the block krylov svd method for use in
+ * Collaborative Fitlering.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#ifndef MLPACK_METHODS_CF_DECOMPOSITION_POLICIES_BLOCK_KRYLOV_SVD_METHOD_HPP
+#define MLPACK_METHODS_CF_DECOMPOSITION_POLICIES_BLOCK_KRYLOV_SVD_METHOD_HPP
+
+#include <mlpack/prereqs.hpp>
+#include <mlpack/methods/block_krylov_svd/block_krylov_svd.hpp>
+
+namespace mlpack {
+
+/**
+ * Implementation of the BLOCK-KRYLOV-SVD policy to act as a wrapper when
+ * accessing BLOCK KRYLOV SVD from within CFType.
```

Comment:
```suggestion
 * Implementation of the Block Krylov SVD policy to act as a wrapper when
 * using Block Krylov SVD for the decomposition type of CF.
```
Sorry to be so pedantic; block Krylov isn't an acronym so there is no need to capitalize it all.